connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConfig.java

```diff
@@ -101,7 +101,11 @@ public class MirrorSourceConfig extends MirrorConnectorConfig {
     public static final Class<?> CONFIG_PROPERTY_FILTER_CLASS_DEFAULT = DefaultConfigPropertyFilter.class;
 
     public static final String OFFSET_LAG_MAX = "offset.lag.max";
-    private static final String OFFSET_LAG_MAX_DOC = "How out-of-sync a remote partition can be before it is resynced.";
+    private static final String OFFSET_LAG_MAX_DOC = "Determines the maximum allowed lag between the source and target partitions before MirrorMaker emits an 'offset sync' event to catch up the remote partition. The lag is calculated as the difference between the latest offset in the source partition, and the last sync emitted to the offset syncs topic. " +
+            "When the lag for a remote partition exceeds the <code>offset.lag.max</code> value, MirrorMaker emits an 'offset sync' event to the 'offset-syncs' topic, which is then used by the <code>MirrorCheckpointTask</code> to translate source and target partitions." +
+            "Setting <code>offset.lag.max</code> to a lower value can be beneficial in scenarios where partitions have consistently low throughput, and the fixed sync interval ('offset.flush.interval.ms') is not acceptable. A lower value will trigger more frequent offset syncs, keeping the target partitions more closely in sync with the source for these low-throughput partitions. " +
```

Comment:
The only part of this line that is worth including is "A lower value will trigger more frequent offset syncs".
What I meant in the previous comment was that 100 is already so close to 0 that nobody will realistically lower this value, so we don't need to describe it in much detail.